subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/runner/TestRunnerImplementation.kt

```diff
@@ -21,8 +20,8 @@ class TestRunnerImplementation(
 
     private val logger = loggerFactory.create<TestRunner>()
 
-    override suspend fun runTests(tests: List<TestRunRequest>, scope: CoroutineScope): TestRunnerResult {
-        return withContext(scope.coroutineContext) {
+    override suspend fun runTests(tests: List<TestRunRequest>): TestRunnerResult {
+        return coroutineScope {
```

Comment:
Suddenly, we have the opportunity to use `coroutineScope` in `suspend` functions
It reduces a little of boilerplate